src/main/webapp/index.html

```diff
@@ -23,8 +23,20 @@ <h1>My Portfolio</h1>
                 <input type="file" accept="image/*" name="image">
                 <button>Upload</button>
             </form>
+            <form action="/image-analysis" method="POST" enctype="multipart/form-data">
```

Comment:
There's two forms here. (Three technically) Is this intended?

Reply:
No, I guess it happened at some point when pulling in changes or something. I made a form for testing the image analysis feature before I had any other changes you guys made to the index.html. But now that the image analysis is in the same servlet as the rest of the stuff, there is only a need for one of course.

Reply:
Cool, that can be removed. Just need to make a note.